test.sh

```diff
@@ -18,9 +18,9 @@ fi
 
 while [[ $# -gt 0 ]]; do
 	case $1 in
-		-a|--appsec)
-			tags="$TAGS appsec"
-			export DD_APPSEC_ENABLED=true
+		--no-appsec)
+			tags="$tags datadog.noasm"
+			export DD_APPSEC_ENABLED=false
```

Comment:
I would keep it simple given this script's goal of quickly running the tests locally by only doing:
```suggestion
		-a|--appsec)
			export DD_APPSEC_ENABLED=true
```

Also, DD_APPSEC_ENABLED is false by default so this would be missing the AppSec enabled case here it seems.